app/views/admin/api_docs/services/edit.html.erb

```diff
@@ -1,3 +1,4 @@
+<% content_for :sublayout_title, 'ActiveDocs' if @service.present? %>
```

Comment:
I definitely have to improve the description of what this PR does because you are not the only person confused with it 👍 so I will improve it and then link here to it. But if you want a fast answer: Because an api_doc can be under the service scope if it has a service or under no service scope if there is no service.